pkgs/build-support/rust/default.nix

```diff
@@ -36,21 +37,22 @@ let
       cargoDepsCopy="$sourceRoot/${cargoVendorDir}"
     '';
 
+  ccForBuild="${buildPackages.stdenv.cc}/bin/${buildPackages.stdenv.cc.targetPrefix}cc";
```

Comment:
The build cc is almost never prefixed. I suppose there could be some weird case where it needs to be though.